src/main/resources/io/jenkins/plugins/designlibrary/Layouts/twoColumn.jelly

```diff
@@ -0,0 +1,24 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
+  <l:layout title="Heading">
```

Comment:
Is it required to use the title in `layout` and `app-bar`? And where is it shown?

Reply:
The `title` attribute is used for the title in the browser tab, the app bar is used at the top of the page as the main heading.

Reply:
Ok, that makes sense. I see the header in the browser tab, but not on the page:

![Bildschirmfoto 2022-07-22 um 16 21 27](https://user-images.githubusercontent.com/503338/180459461-4286539e-af7c-4864-8385-da4d70c13586.png)

Reply:
I need to update Jenkins to support the headings still, it'll be good though as it should standardise the way we do headings on Jenkins.